src/main/java/ObjectPool.java

```diff
@@ -0,0 +1,81 @@
+import vi.viredis.client.ViRedis;
+
+import java.io.IOException;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedTransferQueue;
```

Comment:
Suggestion: if you add a regular, abstract or interface class adorn these with appropriate documentation so that others (and you yourself in the future) know why that class is there. So specify purpose and responsibilities of the class and what you expect from the children of this class (in case of abstract and interface classes).

it helps managing expectations and clarifies the class structure when no other documentation is at hand.

Reply:
done!